src/components/signup/SignupState.js

```diff
@@ -601,17 +597,15 @@ const Signup = ({ navigation }: { navigation: any, screenProps: any }) => {
   }
 
   useEffect(() => {
-    const curRoute = navigation.state.routes[navigation.state.index]
-
     if (state === initialState) {
       return
     }
 
-    if (curRoute && curRoute.key === 'SignupCompleted') {
```

Comment:
could you check for 'last route step' condition ?
e.g. if (navigation.state.index === routes.length - 1) ?

Reply:
fixed